### DISCUSSION TITLE
web server deep link

### DISCUSSION THREAD

**Discussion by beekboss** [serverpod/serverpod#2926] [Q&A]
how can use web server of deep link route ?
no details in documentation .

**Comment by Isakdl** [comment]
Could you expand a little what you are looking for?

**Reply by beekboss** [reply]
I look for share internal page link , like /event/:id , to open app and this page

**Reply by Isakdl** [reply]
What do you mean by "and this page?" Do you expect a web page on the same route? 

For A deeplink in the Flutter app, there is nothing special needed from a Serverpod perspective, you just need to do the normal flutter integration. 

If you want to serve a webpage on the same route you can do that with serverpod, see the docs on the web server here: https://docs.serverpod.dev/concepts/webserver

**Reply by beekboss** [reply]
thanks , but where to set apple-app-site-association file and what route have to use
